.rubocop.yml

```diff
@@ -1,22 +1,3 @@
-AbcSize:
```

Comment:
There's a single change that would need to be made in order to remove this file. This file is still read by Chefstyle since it's just base rules for Rubocop.